index.js

```diff
@@ -153,6 +164,49 @@
 
     zs_info('Einer von uns!');
 
+    // Override setTimeout to allow getting the time left
+    const _setTimeout = setTimeout; 
+    const _clearTimeout = clearTimeout; 
+    const zs_allTimeouts = {};
+    
+    setTimeout = (callback, delay) => {
+        let id = _setTimeout(callback, delay);
+        zs_allTimeouts[id] = Date.now() + delay;
+        return id;
+    };
+
+    clearTimeout = (id) => {
+        _clearTimeout(id);
+        zs_allTimeouts[id] = undefined;
+    }
+    
+    const getTimeout = (id) => {
+        if (zs_allTimeouts[id]) {
+            return Math.max(
+                zs_allTimeouts[id] - Date.now(),
+                0 // Make sure we get no negative values for timeouts that are already done
+            )
+        }
+
+        return NaN;
+    }
+
+    setInterval(() => {
+        const theTimeout = getTimeout(placeTimeout)
+        if (theTimeout == NaN) {
```

Comment:
Dieses if ist nie false! Bitte aufpassen NaN != NaN. Für so etwas verwendet man isNaN()

Reply:
Danke!

Reply:
> Dieses if ist immer false! Bitte aufpassen! NaN != NaN.
> 
> Für so etwas, verwendet man isNaN().

wobei das auch riskant ist, weil `isNan("dsd")` auch true ist, besser `Number.isNaN()`

in diesem Fall aber vermutlich (nicht sicher) egal.

Reply:
Ist tatsächlich egal, trotzdem behoben.